pkg/resourcemonitor/podresourcesscanner.go

```diff
@@ -61,7 +63,8 @@ func (resMon *PodResourcesScanner) Scan() ([]PodResources, error) {
 	//Pod Resource API client
 	resp, err := resMon.podResourceClient.List(ctx, &podresourcesapi.ListPodResourcesRequest{})
 	if err != nil {
-		return nil, fmt.Errorf("Can't receive response: %v.Get(_) = _, %v", resMon.podResourceClient, err)
+		prometheus.UpdateApiCallsFailureMetric(err)
+		return nil, fmt.Errorf("can't receive response: %v.Get(_) = _, %v", resMon.podResourceClient, err)
```

Comment:
we need to record the errors returned by `GetAllocatable` (pkg/resourcemonitor/resourcemonitor.go:188)